### PR TITLE
chore: upgrade OpenFin Workspace

### PR DESCRIPTION
**Pull request by algreasley**
Tried WS v16 .. [here](https://cdn.openfin.co/versions/?product=Workspace#16.1.7)

> Workspace 16.1.7
> Notifications 2.2.1
> Runtime 33.116.77.11

_The below did not work for local dev_ .. EPERM errors on Mac (ARM) after first attempt, which is v slow due to new runtime download - no progress tracking on that , btw. First successful run can be repeated by deleting relevant runtimes under `~/OpenFin/Runtime`

_Stable, from [here](https://cdn.openfin.co/versions/?product=Workspace#18.0.10):_
> Workspace 18.0.10
> Notifications 2.5.1
> Runtime 36.122.80.11

_Latest, from [here](https://cdn.openfin.co/versions/?product=Workspace#19.0.3):_

> Workspace 19.0.3
> Notifications 2.6.2
> Runtime 38.126.82.61

### CODE REVIEW

**Review by bruceharris — DISMISSED** [review]
Looks pretty straightforward. I did not dive in deep, just did a cursory review. If you think a more rigorous review is needed, please LMK I can carve out a bit more time. Thanks @algreasley !

File: packages/client/src/workspace/dock.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const registerDock = () => {
     ],
   }
 
-  Dock.register(dockProvider)
+  await Dock.register(dockProvider)
```
Comment (algreasley) [inline packages/client/src/workspace/dock.ts:40]: surprisingly crucial bit .. without this, it just won't start up
OpenFin must have fixed a bug in their async behaviour, as this code has been like this for ages .. 🙄

File: packages/client/package.json [hunk 2]
```diff
@@ -71,16 +71,15 @@
   "dependencies": {
     "@adaptive/hydra-platform": "4.57.0",
     "@finos/fdc3": "^1.2.0",
-    "@openfin/core": "31.75.4",
-    "@openfin/workspace": "13.0.7",
-    "@openfin/workspace-platform": "13.0.7",
+    "@openfin/core": "33.77.11",
```
Comment (algreasley) [inline packages/client/package.json:74]: not much of a bump in the runtime, but should be enough to cure some of the long-standing OF bugs that we had.
going to latest WS/Runtime was problematic when running up locally on Mac - EPERM error on second run once the runtime has been downloaded .. WS 16 / Runtime 33 does not suffer from this. Subject of (yet) another OF support issue soon ....
At least this / explicit register() fixes the dual Notification Services .. and the out-of-control errors when shutting down!

File: packages/client/package.json [hunk 3]
```diff
@@ -71,16 +71,15 @@
...
+    "@openfin/workspace": "16.1.7",
+    "@openfin/workspace-platform": "16.1.7",
     "@react-rxjs/core": "^0.10.3",
     "@react-rxjs/utils": "^0.9.5",
     "d3": "7.9.0",
     "date-fns": "^2.19.0",
     "downshift": "^6.1.9",
     "lodash": "4.17.21",
-    "openfin-notifications": "1.23.0",
```
Comment (algreasley) [inline packages/client/package.json:83]: now part of Workspace package ..

File: packages/client/vite.config.ts [hunk 4]
```diff
@@ -21,8 +21,8 @@ type BuildTarget = "web" | "openfin" | "finsemble"
 const localPort = Number(process.env.PORT) || 1917
 const showOpenFinProvider = !!process.env.OPENFIN_SHOW_PROVIDER
 
-const OPENFIN_RUNTIME = "29.108.73.14"
-const WORKSPACE_OPENFIN_RUNTIME = "31.112.75.4"
+const OPENFIN_RUNTIME = "33.116.77.11"
```
Comment (algreasley) [inline packages/client/vite.config.ts:24]: could drop the duplication now, I guess .. but we may need to diverge again 🤷🏻

File: packages/client/package.json [hunk 5]
```diff
@@ -61,7 +61,7 @@
     "finsemble:build": "cross-env TARGET=finsemble vite build",
     "storybook": "storybook dev -p 6006",
     "storybook:build": "cross-env STORYBOOK=true storybook build -o ./dist/storybook",
-    "_e2e:openfin:run": "cross-env-shell of-automation $npm_config_manifest_url ./e2e/openfin.spec.js --chromeDriverOverride=122 --devToolsPort=9091 --closeRuntime=never",
+    "_e2e:openfin:run": "cross-env-shell of-automation $npm_config_manifest_url ./e2e/openfin.spec.js --chromeDriverOverride=116 --devToolsPort=9091 --closeRuntime=never",
```
Comment (algreasley) [inline packages/client/package.json:64]: forgot to adjust the chromedriver version again .. this is a pain, so I might comment the OF version numbers in vite.config.ts unless there is a way of syncing them up (I couldn't find one before)

Comment (algreasley) [reply packages/client/package.json]: fixed now .. removing the override is now working ok (it was certainly not previously), taking the chromium version from the specified runtime version

**Comment by bruceharris** [comment]
Took another pass through the recent changes, again, not a deep dive, but everything looks reasonable to me based on my cursory review. Nice work!

**Review by bruceharris — APPROVED** [review]

**Pull request merged**